vars/buildFailureMessage.groovy

```diff
@@ -0,0 +1,17 @@
+@NonCPS
```

Comment:
What's this?

Reply:
so .each methods interpreter adds some overhead, so we should add @NonCPS to bypass the CPS transformation inside that method also when we add @NonCPS on a method, Jenkins will execute the entire method in one go without the ability to pause.

Reply:
Are we sure we want to do that, given this method can take very long to execute?